jetty-core/jetty-openid/src/main/java/org/eclipse/jetty/security/openid/OpenIdConfiguration.java

```diff
@@ -50,7 +50,7 @@ public class OpenIdConfiguration extends ContainerLifeCycle
     private final String clientId;
     private final String clientSecret;
     private final List<String> scopes = new ArrayList<>();
-    private final String authMethod;
+    private final String authenticatorName;
```

Comment:
Ditto.
This file has numerous occurrences of this to fix.

Reply:
@lachlan-roberts ditto ditto ditto ditto